CMakeLists.txt

```diff
@@ -397,6 +397,7 @@ list(
    console
    tracing
    sysfs
+   mem
```

Comment:
`mem` doesn't look like the right name for this module, as it doesn't expose `/dev/mem`. Can you rename it to `null` or `zero` ?

Reply:
Thanks, fixed!